src/init.sh

```diff
@@ -28,6 +32,17 @@ function init_kw()
     if [[ -n "${options_values['arch']}" ]]; then
       sed -i -r "s/(arch=)x86_64/\1${options_values['arch']}/" "$PWD/$KW_DIR/$name"
     fi
+
+    if [[ -n "${options_values['remote']}" ]]; then
+      user=$(get_based_on_delimiter "${options_values['remote']}" "@" 1)
+      remote=$(get_based_on_delimiter "${options_values['remote']}" "@" 2)
+      ip=$(get_based_on_delimiter "$remote" ":" 1)
+      port=$(get_based_on_delimiter "$remote" ":" 2)
```

Comment:
This is the deprecated way to get these sorts of info, `populate_remote_info` and `remote_parameters[..]`